### ISSUE TITLE
Add country and region name translations

### ISSUE THREAD

**Issue by lenafm** [datasketch/geodata#13]

**Comment by jpmarindiaz** [comment]
You can use here the dataset I prepared with country name translations. Remember the coronavirus scripts you used for shi18ny documentation? https://coronavirus.datasketch.co/datasets/tidy-csse-langs/

**Comment by lenafm** [comment]
Yes and the data I was using to create the meta data and topojson for the adm1 regions in each world country has translations for all the adm1 regions.
